editor/app/build.gradle

```diff
@@ -21,7 +21,10 @@ android {
         }
     }
 }
-
+repositories {
```

Comment:
I think you don't need to add these because they're already in [`/editor/build.gradle`](https://github.com/ubclaunchpad/soil-topARgraphy/blob/80544a9f5248115a8ee7c3af530b1bf33dd403c0/editor/build.gradle#L19)